SUPPORT.md

```diff
@@ -0,0 +1,21 @@
+# **Support**
+
+Welcome to our project! We're here to help you with any questions or issues you may encounter while using our software. Here are some ways to get in touch with us:
+
+## **Issue Tracker**
+
+If you find a bug or have a feature request, please create an issue in our **[issue tracker](https://github.com/yourusername/yourproject/issues)**. When creating an issue, please provide as much detail as possible about the problem or feature request. This will help us address the issue or request as quickly and efficiently as possible.
```

Comment:
Fix the url?

Reply:
Thanks for the comment, I will modify it.